lib/plugins.js

```diff
@@ -519,7 +519,7 @@ function plugins (settings, deviceId, schema) {
           return Promise.all(cryptoCodes.map(cryptoCode => {
             return exchange.fetchExchange(settings, cryptoCode)
               .then(exchange => ({
-                fiatCode: exchange.account.currencyMarket,
+                fiatCode: exchange.account?.currencyMarket,
```

Comment:
This produces a harmless error, right? Allowing for null here is potentially harmful. It's also not related to the PR so you can just undo the change here.